lib/streamlit/elements/widgets/chat.py

```diff
@@ -515,6 +515,7 @@ def chat_input(
         )
 
         chat_input_proto.file_type[:] = file_type if file_type is not None else []
+        chat_input_proto.max_file_size_mb = config.get_option("server.maxUploadSize")
```

Comment:
nit: for consistency it might be good to use the same proto property name as in file uploader: `max_upload_size_mb`